### PR TITLE
Backgorund running

### PR DESCRIPTION
**Pull request by serqiiiio**
I changed the 16th line of index.js file  to  
"const game = new Phaser.Game(config);"
`

### CODE REVIEW

**Comment by serqiiiio** [comment]
One more thing is normal that the background is weird becuase we put the background in the origin (0,0)

**Review by edwinPato — APPROVED** [review]

**Review by virichino — APPROVED** [review]

File: background.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { Plataform } from "./plataform";
+export class Background extends Phaser.Scene {
+  preload() {
+    this.load.image("background", "images/background.jpg");
+    this.load.image("plataforma", "images/plataforma.png");
```
Comment (J0c4) [inline background.js:5]: Please use english for all text/code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="”en”">
+  <head>
+    <meta charset="”UTF-8″" />
+    <title>Document</title>
+  </head>
+  <body></body>
+  <h1>Hola</h1>
```
Comment (J0c4) [inline index.html:8]: Please remove this line if not needed

**Comment by serqiiiio** [comment]
Team changed version oso for that I'm closing the pull request

**Pull request closed without merging**